static/src/javascripts/projects/common/modules/commercial/dfp-ophan-tracking.js

```diff
@@ -0,0 +1,88 @@
+define(['raven', 'common/utils/detect'], function (raven, detect) {
+
+    function attachListeners(googletag) {
+        var adTimings = {};
+        spyOnAdDebugTimings();
+        reportAdsAndTimingsOnRender();
```

Comment:
As the function calls are statements, rather than more variables, wouldn't they align with the var itself? I might be misunderstanding here.